perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/beans/ValueTransformerBase.java

```diff
@@ -0,0 +1,57 @@
+package cz.metacentrum.perun.ldapc.beans;
+
+import java.util.Collection;
+import java.util.Map;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.ldapc.model.AttributeValueTransformer;
+
+public class ValueTransformerBase implements AttributeValueTransformer {
+	
+	protected Boolean massTransformationPreferred;
+	protected Boolean reduce;
```

Comment:
These variables are not initialized and when you use getter result in IF condition, it will throw ``NullPointerException``. Since the LDAPc doesn't fail, it seems, that it is not used (getters are sometimes overriden in specific transformers), but some default value would be better.

Reply:
Initialized to false.